src/components/addSourceWizard/FinalWizard.js

```diff
@@ -195,21 +195,24 @@ const FinalWizard = ({
     );
   }
 
+  const appendTo = React.useMemo(() => document.querySelector('.pf-c-page.chr-c-page'));
```

Comment:
You are missing dependency array
```suggestion
  const appendTo = React.useMemo(() => document.querySelector('.pf-c-page.chr-c-page'), []);
```

Reply:
Good catch, thanks!